src/Components/AddTask.jsx

```diff
@@ -1 +1,46 @@
+import React , { useState} from "react";
+import Button from "./Button";
 
+function AddTask({ addTask }) {
+    const [taskName, setTaskName] = useState("");
+    const [status, setStatus] = useState("Todo");
+
+    const taskStatuses = ["Todo", "In-Progress", "Completed"];
+    
+    const createClick = () => {
+        if(taskName.trim()) {
+            addTask({task: taskInput, status});
+            setTaskName(""); // clear after
+            setStatus("Todo");
+        }
+    }
+    return (
+        <div className="todo-input">
+            <input 
+                type="text" 
+                placeholder="Enter todo" 
+                value={taskName} onChange={(e) => setTaskName(e.target.value.trim())} maxLength={25}>
```

Comment:
It is better to avoid using anonymous functions.

```
const onTodoInputChange = (event) => {
  setTaskName(event.target.value.trim());
}

onChange={onTodoInputChange}
```